packages/client/hmi-client/src/model-representation/petrinet/nested-petrinet-renderer.ts

```diff
@@ -19,10 +20,13 @@ const { getNodeTypeColor } = useNodeTypeColorPalette();
 export class NestedPetrinetRenderer extends PetrinetRenderer {
 	nestedMap: any;
 
+	transitionMatrices: any;
```

Comment:
Slightly better typing would be appreciated, now that we have fletched out actual data-structures.

Reply:
let me know if the new typings look good to you